docs/installation/upgrade/include/2.3-to-2.4.inc

```diff
@@ -56,6 +56,18 @@ passdb some_name {
 | `imap_id_log` setting | Replaced by the [[event,imap_id_received]] event. |
 | size.virtual | size.virtual field is no longer written to dovecot.index.cache file as it is duplicating vsize record in dovecot.index file. Reading of the field from old files is supported. |
 | SETQUOTA / `quota_set` | Quota limits can no longer be modified using the IMAP SETQUOTA command. |
+| `namespace/alias_for` | [[setting,namespace_alias_for ]] |
+| `namespace/disabled` | [[setting,namespace_disabled ]] |
+| `namespace/hidden` | [[setting,namespace_hidden ]] |
+| `namespace/ignore_on_failure` | [[setting,namespace_ignore_on_failure ]] |
+| `namespace/inbox` | [[setting,namespace_inbox ]] |
```

Comment:
I am not against long lists. Only against unknown information. If this is technically not removed and the rest of the documentation makes it clear, that this is still supported but not usually used, I shall remove these entries from this list.

Reply:
"Removed" sounds like the feature is completely removed. These and other similar setting changes are more "renamed settings", so I think they belong to a different list.

Reply:
I suggest I create a ticket for this, because I don't want to lose focus on the task at hand. And maybe the replacement/rename-ment list is better suited after the most changes are comitted.